gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/data/repository/AvatarRepository.kt

```diff
@@ -4,30 +4,42 @@ import android.net.Uri
 import androidx.core.net.toFile
 import com.gravatar.quickeditor.data.models.QuickEditorError
 import com.gravatar.quickeditor.data.storage.TokenStorage
+import com.gravatar.quickeditor.ui.avatarpicker.copy
 import com.gravatar.restapi.models.Avatar
 import com.gravatar.services.AvatarService
+import com.gravatar.services.ErrorType
 import com.gravatar.services.GravatarResult
 import com.gravatar.types.Email
 import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.flow.asSharedFlow
 import kotlinx.coroutines.withContext
 
 internal class AvatarRepository(
     private val avatarService: AvatarService,
     private val tokenStorage: TokenStorage,
     private val dispatcher: CoroutineDispatcher,
 ) {
+    fun getAvatarsFlow(email: Email) = avatarsFlow(email).asSharedFlow()
```

Comment:
```suggestion
    fun getAvatars(email: Email) = avatarsFlow(email).asSharedFlow()
```

⛏️ I think I would shorten the name and skip the `flow` suffix. The return type is specified so not sure having it in the name adds much value.